core/src/main/java/org/apache/sedona/core/spatialRDD/SpatialRDD.java

```diff
@@ -129,9 +138,24 @@
      * The sample number.
      */
     private int sampleNumber = -1;
-
-    public int getSampleNumber()
-    {
+    /**
+     * Geometry Type Defaults to Geometry Collection
+     */
+    private GeometryType geometryType;
+    
+    public SpatialRDD() {
```

Comment:
Is there a reason why you need to add geometry type to SpatialRDD? Because you need to use this information to define the type of geospatial objects in a Parquet file?

I don't think this is a good idea. Currently, this generic SpatialRDD is not bounded to a particular type. It allows mixed spatial objects in an RDD. E.g.,

POINT (XXX)
POLYGON (XXX)
MULTIPOLYGON (XXX)
POINT (XXX)

This scenario is actually quite common in real world. A WKT file of state boundaries or country boundaries may consist of both POLYGON and MULTIPOLYGON.